src/services/authentication.jsx

```diff
@@ -10,17 +10,21 @@ export const authenticationAPI = createApi({
   endpoints: (builder) => ({
     getSignIn: builder.mutation({
       query: ({ data }) => {
+        let formData = new FormData();
```

Comment:
`formData` does not seem to be updated anywhere else in the scope. Can we use `const` ?